speech_recognition/__init__.py

```diff
@@ -1390,7 +1393,37 @@ def recognize_tensorflow(self, audio_data, tensor_graph='tensorflow-data/conv_ac
             for node_id in top_k:
                 human_string = self.tflabels[node_id]
                 return human_string
+            
+    def recognize_vosk(self, audio_data, language='en'):
+        from vosk import Model, KaldiRecognizer
+
+        if not os.path.exists("model"):
+            return "Please download the model from https://github.com/alphacep/vosk-api/blob/master/doc/models.md and unpack as 'model' in the current folder."
+            exit (1)
+
+        import pyaudio
+
+        rec = KaldiRecognizer(modelVosk, 16000)
```

Comment:
You'd better use something like this:

```
if not hasattr(self, 'vosk_model'):
     self.vosk_model = Model()

rec = KaldiRecognizer(self.vosk_model, 16000);
```

Reply:
Oooof.
I have a ton of work to do and like a billion tabs opened... :laughing: 
Will do it as soon as possible

Reply:
Done